pkg/apis/apis/v1alpha1/permissionclaims/labels.go

```diff
@@ -37,3 +37,50 @@ func ToLabelKeyAndValue(permissionClaim apisv1alpha1.PermissionClaim) (string, s
 	labelKeyHashLength := validation.LabelValueMaxLength - len(apisv1alpha1.APIExportPermissionClaimLabelPrefix)
 	return apisv1alpha1.APIExportPermissionClaimLabelPrefix + hash[0:labelKeyHashLength], hash, nil
 }
+
+func ValidateClaim(group, resource, identityHash string, grsToBoundResource map[apisv1alpha1.GroupResource]apisv1alpha1.BoundAPIResource) (bool, error) {
```

Comment:
have a hard time to follow the logic here. `grsToBoundResource` is the union of all bound resourcses in a workspace?

This needs go doc.

Reply:
What does "valid" mean as a concept? Valid in which context, for which use?

Reply:
updated

Reply:
My comment stands. IMO this function does not make sense as a generic helper. It actually is something like `IsBoundResourceOrIdentityIsEmpty`. This kind of helper should be local to the code using it. It's not a good abstraction.